static/app/utils/profiling/hooks/useProfileFilters.tsx

```diff
@@ -0,0 +1,31 @@
+import {useEffect, useState} from 'react';
+
+import {Client} from 'sentry/api';
+import {Organization, Tag} from 'sentry/types';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+type ProfileFilters = {[key: string]: Tag};
```

Comment:
```suggestion
type ProfileFilters = Record<string, Tag>;
```
Tbh, you could just drop this type and just type fn return here

Reply:
I added it to type the `useState` below as well.